cmd/tealdbg/local_test.go

```diff
@@ -1459,3 +1460,307 @@ func TestGroupTxnIdx(t *testing.T) {
 	a.NoError(err)
 	a.True(pass)
 }
+
+func TestRunAllGloads(t *testing.T) {
+
+	partitiontest.PartitionTest(t)
+	a := require.New(t)
+
+	sourceA := `#pragma version 6
+
+	txn ApplicationID
+	bz handle_createapp
+	
+	int 99
+	store 1
+	
+	itxn_begin
+		int acfg
+		itxn_field TypeEnum
+		int 1000000
+		itxn_field ConfigAssetTotal
+		int 3
+		itxn_field ConfigAssetDecimals
+		byte base64 AA== 
+		itxn_field ConfigAssetUnitName
+		byte base64(AAAAAAAAAAA=)
+		itxn_field ConfigAssetName
+		pushbytes 0x0000000000000000 
+		itxn_field ConfigAssetURL
+		global CurrentApplicationAddress
+		dup
+		dup2
+		itxn_field ConfigAssetManager
+		itxn_field ConfigAssetReserve
+		itxn_field ConfigAssetFreeze
+		itxn_field ConfigAssetClawback
+	itxn_submit
+	
+	handle_createapp:
+	int 1`
+
+	sourceB := `#pragma version 6
+
+	txn ApplicationID
+	bz handle_createapp
+	
+	gload 2 1
+	itob
+	log
+	
+	handle_createapp:
+	int 1`
+
+	ops, err := logic.AssembleString(sourceA)
+	a.NoError(err)
+	progA := base64.StdEncoding.EncodeToString(ops.Program)
+
+	ops, err = logic.AssembleString(sourceB)
+	a.NoError(err)
+	progB := base64.StdEncoding.EncodeToString(ops.Program)
+
+	// Transaction group with 5 transactions
+	// 1. Payment txn to app A
+	// 2. Payment txn to app B
+	// 3. App call to app A
+	// 4. App call to app B with gload on app A scratch slot
+	ddrBlob := `{
```

Comment:
@algochoi Beginner's question:  How is `ddrBlob` generated?

* I'm asking for my edification and also to consider how a future maintainer will make edits.
* I realize the highlighted content is `dryrun` output.  Though it's not entirely clear to me what initial state was provided.

Reply:
I dumped the dryrun output (as a JSON instead of msgp) like: [this](https://github.com/algochoi/algorand-onboard/blob/a80085f336f8d79606374a961e69d5268b360257/tealdbg-tutorial/tealdbg_deploy.py#L215). 

I created the apps beforehand using a `ApplicationCreateTxn`, and then created a group transaction with 4 txns annotated by the comment above.

Reply:
@algochoi Thanks for the explanation and for a quick verbal.  I'm good to resolve.

For completeness -  I have lingering concerns about the difficulty in maintaining tests that require manual setup to reconstruct test input.  Though it seems the convention is applied elsewhere, so let's run with it for now.